### DISCUSSION TITLE
Cycle start date not taken into account

### DISCUSSION THREAD

**Discussion by codams** [community/community#8944] [Projects and Issues] [answered]
Hi everyone, 

I have a minor issue/question about the new **cycling field** : 
1. I went to the project settings and wanted to create cycles that match our team sprints (classic 2 weeks) but wanted to start on next monday (19 dec.) since we end our current sprint this friday.
2. So I set the start date as this :
![image](https://user-images.githubusercontent.com/7759894/146356954-0703c923-8ce3-4503-9cb3-cff6d137181f.png)
3. And click on `Add iteration`, then the following iteration is created : 
![image](https://user-images.githubusercontent.com/7759894/146357154-c806f8f7-e6e3-4baf-a2c2-519f3f33e67e.png)
4. Starting today, it is not really blocking since I will just wait until monday to create all the iterations 🐙 

edit.
Just for reference the infos sent when we create an iteration : 

![image](https://user-images.githubusercontent.com/7759894/146361290-14cce4ea-77b3-4405-a216-6bb16c13f1f7.png)

Happy holiday season y'all ! ⭐

**Comment by mlangenberg** [comment, marked as answer]
I believe this should be labeled as a bug, as it used to work before. I have created several iterations with 'gaps' a few weeks ago and today I'm unable to do anymore.

**Reply by mlangenberg** [reply]
Seems to be working again. 🙌 

![image](https://user-images.githubusercontent.com/992/147229689-70d2e2e5-2f2f-483f-ad67-85fc21f5558e.png)

**Reply by codams** [reply]
Niiiiice
